<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transprot</class>
 <widget class="QDialog" name="transprot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="titre">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>931</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="title">
    <string>GESTION DE TRANSPORT : </string>
   </property>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>921</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>481</height>
      </rect>
     </property>
     <attribute name="label">
      <string>moyens de transport</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>551</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Ajouter moyen</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_ref">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>380</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="reference">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>150</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>model :</string>
        </property>
       </widget>
       <widget class="QLabel" name="nom">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>120</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>ref:</string>
        </property>
       </widget>
       <widget class="QLabel" name="quantite">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>190</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>confort :</string>
        </property>
       </widget>
       <widget class="QLabel" name="quantite_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>250</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>nombres de places :</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_np">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>250</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="quantite_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>300</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>climatisation :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdi_confort">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>200</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cli">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>290</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>358</x>
          <y>150</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>VOITURE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BUS</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Afficher moyen</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>501</width>
          <height>301</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_mod">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>50</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supp">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_sup">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ref_2">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>140</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdi_confort_2">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>230</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_np_2">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>280</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cli_2">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>330</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="quantite_4">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>280</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>nombres de places :</string>
        </property>
       </widget>
       <widget class="QLabel" name="nom_9">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>150</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>ref:</string>
        </property>
       </widget>
       <widget class="QLabel" name="reference_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>180</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>model :</string>
        </property>
       </widget>
       <widget class="QLabel" name="quantite_5">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>220</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>confort :</string>
        </property>
       </widget>
       <widget class="QLabel" name="quantite_6">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>330</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>climatisation :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Refreche">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>410</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Refreche</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_rechercher">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Rechercher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ref</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>modele et confort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>modele et nombre_de_place et confort</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_trier">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>50</y>
          <width>73</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Trier </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ref</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>modele</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>modele et nombre de place </string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>718</x>
          <y>190</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>VOITURE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BUS</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>réclamation :</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>481</height>
      </rect>
     </property>
     <attribute name="label">
      <string>gestion abonnements</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>681</width>
        <height>391</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ajouter abonné</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nom_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>70</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLabel" name="nom_3">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>NOM:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>120</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nom_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>170</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>prenom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="nom_5">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>ref moyen :</string>
        </property>
       </widget>
       <widget class="QLabel" name="nom_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>260</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>260</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nom_7">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>260</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>jusqu'au :</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>260</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nom_8">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>310</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>payer :</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>310</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
